modules/ROOT/pages/manage-proxy.adoc

```diff
@@ -88,15 +99,18 @@ sudo /opt/anypoint/runtimefabric/rtfctl apply http-proxy '' --no-proxy ''
 -----
 
 === Configure a Proxy for the API Manager Agent
-Add the following properties to the application:
+
+To configure a proxy for the API Manager Agent, add the following properties to the application:
```

Comment:
@PabloCamino I'm not sure this sentence makes sense. "...add the following properties to the application." Which application?